### ISSUE TITLE
A few compat issues/suggestions with some other mods

### ISSUE THREAD

**Issue by ChromaPIE** [iwoplaza/MoBends#83]
**Silent's Gems**: Error in armor models of this mod.

**SlashBlade**: Is it possible to add bends compatible with this mod? You can imagine how cool it will be when Mo' Bends, Shoulder Surfing and SlashBlade get together. I would even delete SEKIRO! At least it is suggested to just render the model of the blade in mainhand as it looks like holding nothing in the latest released version.

**Contruct's Armory**: Glitch happens in the preview model in its workbench GUI and the armor rendering issue is the same as one Silent's Gems has.

**Traveller's Backpack** _and other on-body rendered mods_: Hope to see them rendered correctly in the last release of Mo' Bends.

**Comment by bobodoustaud** [comment]
also the terrasteel armor from botania is weird rendered

**Comment by undyingtheglass** [comment]
Yes, when I loaded Mo' Bends with SlashBlade, I used the SA error with the 'eat' animation, and I hope this problem can be solved
